app/models/tax_svc.rb

```diff
@@ -15,7 +15,7 @@ def get_tax(request_hash)
     response = JSON.parse(res.body)
 
     if response['ResultCode'] != 'Success'
-      logger.info_and_debug("Avatax Error: Order ##{request_hash[:DocCode]}", response)
+      logger.info "Avatax Error: Order ##{request_hash[:DocCode]}, Response: #{response}"
```

Comment:
Hi @sebastiandl, it is important this change because the log it is configured with INFO level and the response log within logger.info_and_debug if for DEBUG level

Reply:
Oh you are right! But then how come we see these logs in production?? :thinking: